phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -911,7 +911,10 @@ private void addCoprocessors(byte[] tableName, TableDescriptorBuilder builder, P
                     QueryServicesOptions.DEFAULT_INDEX_REGION_OBSERVER_ENABLED);
 
             if (tableType == PTableType.INDEX && !isTransactional) {
-                if (globalIndexerEnabled && !newDesc.hasCoprocessor(GlobalIndexChecker.class.getName())) {
+
```

Comment:
A bug fix? Should this fix be done in another JIRA and PR?

Reply:
Added new PR: https://github.com/apache/phoenix/pull/522/files